include/functions.h

```diff
@@ -369,7 +369,7 @@ void TitleCard_InitBossName(GlobalContext* globalCtx, TitleCardContext* titleCtx
                             u8 height);
 void TitleCard_InitPlaceName(GlobalContext* globalCtx, TitleCardContext* titleCtx, void* texture, s32 x, s32 y,
                              s32 width, s32 height, s32 delay);
-s32 func_8002D53C(GlobalContext* globalCtx, TitleCardContext* titleCtx);
+s32 TitleCard_ResetTimers(GlobalContext* globalCtx, TitleCardContext* titleCtx);
 void Actor_Kill(Actor* actor);
 void Actor_SetFocus(Actor* actor, f32 offset);
```

Comment:
from looking at its usage, I think a more appropriate name is probably something like `TitleCard_Clear`, or something that makes it clear that it is making the title card go away. it does reset timers, but I think its a bit too literal

its specifically used whenever player tries to do anything involving a get item. the title cards and get items use the same memory for their assets, so player clears/hides the title cards out of view before pulling out something that uses a get item object